pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "fcmeans"
+version = "0.1.0"
+description = ""
+authors = ["Madson Dias <madsonddias@gmail.com>", "dirk <https://github.com/dirknbr>", "José Alberth <jalberth14@gmail.com>"]
+
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/omadson/fuzzy-c-means"
+repository = "https://github.com/omadson/fuzzy-c-means"
+keywords = ["machine-learning", "data-science", "fuzzy-c-means", "clustering"]
+classifiers = [
+    "Operating System :: OS Independent",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+include = [
+    "LICENSE",
+]
+
+[tool.poetry.dependencies]
```

Comment:
I think the only dependence on fuzzy c-means is numpy. You do not need to add other libraries.

Put them as development dependencies.